councilmatic/settings.py

```diff
@@ -8,8 +8,33 @@
 
 env = environ.Env(
     # Set default values
-    DEBUG=(bool, False),
-    LOCAL_DOCKER=(bool, False),
+    LOCAL_DOCKER=(bool, True),
```

Comment:
We default to development settings in other apps. Think it's a good idea to preserve this pattern and keep the size of the local env file small, since these are overridden in deployment already.